sched/scheduler/cluster_state.go

```diff
@@ -37,6 +37,15 @@ type clusterState struct {
 	stats            stats.StatsReceiver           // for collecting stats about node availability
 }
 
+func (c *clusterState) isOfflined(ns *nodeState) bool {
+	for _, v := range c.offlinedNodes {
```

Comment:
isn't this a map?

Reply:
c.offlinedNodes? yeah, the value (v) is a *nodeState though

Reply:
right, I mean, can't we lookup via ns.node.NodeID instead of iterating?

Reply:
ah, gotcha. yep 🤦‍♂